tests/test_views.py

```diff
@@ -1763,7 +1763,7 @@ def test_password_change_invalid_new_password_length(self):
             old_password='password',
             new_password='12345',
             confirm_password='12345',
```

Comment:
Could you also change this so that it's '1234567' so it tests the new edge case? And is there already another test that will try out an 8 character password to verify that that's okay? Might as well get both sides of the edge.